Update README.md (#60)
friendsofgo/killgrave: README.md

## Versions

> Actually, `Killgrave` works using `sem ver` but in 0 version, for that the 'minor' will be changed when some broken changes are added to the application, and the 'patch' would be changed when a new feature with new changes are added or for bug fixing, when v1.0.0 will be released, `Killgrave` will be change to use a normal use of `sem ver`. 

Reviewer: ```suggestion
Actually, `Killgrave` works using [`SemVer`](https://semver.org/) but in 0 version, which means that the 'minor' will be changed when some broken changes are introduced into the application, and the 'patch' will be changed when a new feature with new changes is added or for bug fixing. As soon as v1.0.0 be released, `Killgrave` will start to use [`SemVer`](https://semver.org/) as usual.
```

## Branch master

> The Master branch contains all the latest changes on the application. Therefore, the branch `master` is not a stable version.

Reviewer: ```suggestion
The Master branch contains all the latest changes on the application. Therefore, the `master` branch is not a stable version.
```

> If you want to have the latest changes then you will need to use the branch master, but remember that some breaking changes can be added to this branch.

Reviewer: ```suggestion
If you want to have the latest changes then you will need to use the `master` branch, but remember that some breaking changes can be added to this branch.
```

## Future releases
This project is in continuous improvement so you can check the [Issues](https://github.com/friendsofgo/killgrave/issues) and hopefully give it support or create your own issue.

> Furthermore, on the `branch master` you can find the [CHANGELOG.md](https://github.com/friendsofgo/killgrave/blob/master/CHANGELOG.md) file that contains all the new features that will be added on the next release. 

Reviewer: ```suggestion
Furthermore, on the `master` branch you can find the [CHANGELOG.md](https://github.com/friendsofgo/killgrave/blob/master/CHANGELOG.md) file that contains all the new features that will be added on the next release. 
```

```
Remember to use the [-p](https://docs.docker.com/engine/reference/run/#expose-incoming-ports) flag to expose the container port where the application is listening (3000 by default).

> NOTE: If you want to use `killgrave` through Docker at the same time you must use your own dockerised HTTP-based API, be careful with networking issues.

Reviewer: Not sure about that `must` since it's not a mandatory statement.

It basically tries to say: if your application is dockerised and you want to use Killgrave also through Docker, be careful. But you can use Killgrave through Docker without the need of dockerising your application.

Author: Ahhh I understand, yes that change I made isn't clear, I'll work on that.

Author: OK I've made a change to hopefully make it clearer
